tests/installation/livecdreboot.pm

```diff
@@ -3,6 +3,15 @@ use base "y2logsstep";
 use testapi;
 use bmwqemu ();
 
+sub use_wicked() {
+    type_string "cd /proc/sys/net/ipv4/conf\n";
+    type_string "for i in *[0-9]; do echo BOOTPROTO=dhcp > /etc/sysconfig/network/ifcfg-\$i; wicked --debug all ifup \$i; done\n";
+}
+
+sub use_ifconfig() {
+    type_string "dhcpcd eth0\n";
+}
+
```

Comment:
why are there in tests when they are called from lib/y2logstep? This may not work for other tests based upon y2logstep.